src/rust/engine/options/src/args_tests.rs

```diff
@@ -10,9 +10,7 @@ use crate::{option_id, DictEdit, DictEditAction, Val};
 use crate::{ListEdit, ListEditAction, OptionId, OptionsSource};
```

Comment:
All changes in this test file are either new feature tests (which failed on the old implementation) or replacing direct construction of `Args` with a call to `Args::new()`.